web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
@mandawilson  Yeah I think having the URL commented out is a good idea. It might be good to point to some documentation of how to use the session service, unless it will be an MSKCC internal thing for now. In that case it might be better to add `(mskcc internal)` in the comment